### ISSUE TITLE
Cannot find module 'bourbon-neat'

### ISSUE THREAD

**Issue by GaryJones** [craigsimps/gulp-wp-toolkit#50]
I'm getting the following for all gulp commands in Incipio. I'd just cleaned the yarn cache.

```
module.js:487
    throw err;
    ^

Error: Cannot find module 'bourbon-neat'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.resolve (internal/module.js:18:19)
    at Object.<anonymous> (/Users/gary/Local Sites/incipio/app/public/wp-content/themes/incipio/node_modules/node-neat/index.js:4:30)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
```

**Comment by GaryJones** [comment]
Can we consider dropping `node-neat` and `node-bourbon` from the toolkit, and let themes add that to their `package.json` if they use it?

**Comment by GaryJones** [comment]
#51 does fix this issue.

**Comment by craigsimps** [comment]
Dropping Bourbon & Neat from the toolkit makes sense in terms of keeping it pretty neutral and leaving it open to the consuming theme to make decisions. Leaving normalize in there is fine though. 

I would perhaps like to pass a config object to the  

```includePaths: [].concat(bourbon, neat, normalize)``` 

line from `css.js` which we can then update from the consuming theme's `Gulpfile.js` so other libraries could be injected (Susy for example or Breakpoint SASS.

**Comment by GaryJones** [comment]
Not sure what you mean there, so maybe either merge 51 and then adapt, or update the branch that 51 is based on :-)

**Comment by craigsimps** [comment]
With #51 merged I don't think there will be a way for a consuming theme to add Bourbon & Neat back in unless it rewrites the CSS task in its own `Gulpfile.js`. 

My idea from previous comment would be that we could pass in some config from the consuming theme's `Gulpfile.js`, along the lines of:

```
sassLibraries: [ bourbon, neat ]
```
Which could be added to the `css.js` task, giving something like:

```
.pipe(sass.sync({
    outputStyle: config.css.outputStyle,
    includePaths: [].concat(normalize, config.sassLibraries)
}))
```

**Comment by GaryJones** [comment]
>I don't think there will be a way for a consuming theme to add Bourbon & Neat back

```scss
// style.css
 @import ../../node_modules/node-bourbon/path/to/scss
 @import ../../node_modules/node-neat/path/to/scss
```

?

**Comment by craigsimps** [comment]
Well, that's true. I just liked having `@import "bourbon";` for example and letting the Gulp task swap in the library. Of course, the advantage of directly importing is that mixins will be clickable within the IDE, where they weren't with my method.

**Comment by craigsimps** [comment]
Closing as fixed when #51 was merged.
